src/org/ses/android/soap/RegisterParticipantActivity.java

```diff
@@ -256,16 +287,31 @@ public void onClick(View v) {
     }
 
     public void setCurrentDateOnView() {
-
-        final Calendar c = Calendar.getInstance();
-        year = c.get(Calendar.YEAR);
-        month = c.get(Calendar.MONTH);
-        day = c.get(Calendar.DAY_OF_MONTH);
+        // will attempt to load date from bundle
+        try{
+            String dob = getIntent().getStringExtra("dob").trim();
+            String[] dobSplit = dob.split("/");
+
+            day = Integer.parseInt(dobSplit[0]);
+            month = Integer.parseInt((dobSplit[1]));
+            year = Integer.parseInt(dobSplit[2]);
+
+            edt_dob.setText(new StringBuilder()
```

Comment:
Isn't this code just a duplicate of the code at the bottom of this function? Do you need both or can you get rid of one?